lib/MusicBrainz/Server/EditSearch/Predicate/Role/User.pm

```diff
@@ -91,8 +91,8 @@ role {
         }
     };
 
-    method valid => sub {
-        my ($self) = @_;
+    around valid => sub {
+        my ($orig, $self) = @_;
 
         return 1 unless $self->operator_cardinality($self->operator);
```

Comment:
This line should probably be replaced with `return 0 unless $self->$orig;`, since the base method checks cardinality. (Otherwise, there's no point to using `around`, and `override` should be used instead.)